c2corg_ui/static/js/documentediting.js

```diff
@@ -185,21 +186,20 @@ app.DocumentEditingController = function($scope, $element, $attrs,
    */
   this.api_ = appApi;
 
-  // allow association only new outing  to existing route
+  // allow association only new outing to existing route
   if (this.ngeoLocation_.hasParam('routes')) {
     this.urlParams_ = {'routes': this.ngeoLocation_.getParam('routes')};
     this.pushDocToAssociations_();
   }
 
-  // When creating a new document, the model is not created until
-  // the form is touched. At least create an empty object.
-  this.scope_[this.modelName_] = this.scope_['document'] = this.documentService.document;
```

Comment:
OK I misread that you said the problem was in the SVG file!
By the way do we really need to include the SVG file directly in the page HTML? Can't it be used as a separated file?